x-pack/plugins/code/server/lsp/language_servers.ts

```diff
@@ -38,14 +38,10 @@ export const JAVA: LanguageServerDefinition = {
   languages: ['java'],
   launcher: JavaLauncher,
   installationType: InstallationType.Plugin,
-  installationPluginName: 'javaLanguageServer',
+  installationPluginName: 'java_languageserver',
   installationFolderName: 'jdt',
-  version: '1.0.0-SNAPSHOT',
-  build: '201812040656',
-  downloadUrl: (lang: LanguageServerDefinition) =>
-    `https://github.com/Poytr1/eclipse.jdt.ls/releases/download/v${
-      lang.version
-    }/jdt-language-server-${lang.version}-${lang.build}.tar.gz`,
+  downloadUrl: (lang: LanguageServerDefinition, version: string) =>
+    `http://download.elasticsearch.org/code/java_languageserver/release/java_languageserver-${version}-$OS.zip`,
```

Comment:
nit: can `https://` work?

Reply:
yes let's uss https

Reply:
also, it seems we are using java_langserver in the plugin